bindings/python/setup.py

```diff
@@ -10,9 +10,15 @@
 
 
 HERE = os.path.abspath(os.path.dirname(__file__))
+BUILD_DIR = os.path.join(HERE, 'pymongoarrow')
 
-if platform not in ('linux', 'darwin'):
-    raise RuntimeError("Unsupported plaform {}".format(platform))
+# Find and copy the binary arrow files, unless
+# MONGO_NO_COPY_ARROW_LIB is set (for instance in a conda build).
+COPY_LIBARROW = not os.environ.get("MONGO_NO_COPY_LIBARROW", False)
+
+# Find and copy the binary libbson file, unless
+# MONGO_NO_COPY_LIBBSON is set (for instance in a conda build).
+COPY_LIBBSON = not os.environ.get("MONGO_NO_COPY_LIBBSON", False)
```

Comment:
Can you explain why we don't want to copy libbson (and libarrow) for conda builds? We should also record why we made this decision in the Jira ticket + commit message so we can refer back to it in the future.

Reply:
I'll update the docstring.  It is already documented in the JIRA ticket why shared libraries are handled differently on conda vs wheels.